lib/src/components/NxToast/NxToast.scss

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-toast__wrapper {
+  // nx-toast__wrapper has position absolute to ensure toasts stay in the top right corner of the page.
+  // Using display:flex and flex-end ensures the right side of the toast is aligned with the right side of 
+  // nx-page-main. The nx-toast_container has position fixed applied so it remains fixed while scrolling.
+  position: absolute;
+  top: 0;
+  right: 0;
+  display: flex;
+  justify-content: flex-end;
+  margin-right: var(--nx-spacing-6x);
+}
+
+.nx-toast__container {
```

Comment:
This should probably be `.nx-toast-container`.  It's not contained within `.nx-toast` so making it a BEM element under the `nx-toast` BEM block is iffy.

Reply:
> This should probably be `.nx-toast-container`. It's not contained within `.nx-toast` so making it a BEM element under the `nx-toast` BEM block is iffy.

Updated here: [f2e0e7a](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/f2e0e7a3dfad7f44e96d5bf97d3867303c018707). I also updated wrapper in case we end up leaving the layout of the container as a child of NxPageMain as is.